Core/include/Acts/EventData/SingleBoundTrackParameters.hpp

```diff
@@ -142,13 +142,31 @@ class SingleBoundTrackParameters {
     return m_paramSet.getUncertainty<kIndex>();
   }
 
+  /// Space-time position four-vector.
+  ///
+  /// @param[in] geoCtx Geometry context for the local-to-global transformation
+  ///
+  /// This uses the associated surface to transform the local position on the
+  /// surface to globalcoordinates. This requires a geometry context to select
+  /// the appropriate transformation and might be a computationally expensive
+  /// operation.
+  Vector4D position4(const GeometryContext& geoCtx) const {
```

Comment:
Following your naming, why not `fourPosition`?

Reply:
That could work, too. I prefer `position4` because it is a bit shorter and because it is consistent with existing code in `ActsFatras::{Hit,Particle}`.